### DISCUSSION TITLE
Command input binds

### DISCUSSION THREAD

**Discussion by SparkyDabs** [winstxnhdw/lc-hax#506] [Q&A] [answered]
Hello, Sorry to have to ask this as I'm sure it reveals my inability but I'm trying to learn. I had found it once before but somehow lost the file, where do I find the list that tells me what inputs are what key binds on my keyboard? (Example, "M1") I know "M1" references to a keypress or series of them but can't seem to find the list of these. Any help is appreciated, hope you have a wonderful day

**Comment by winstxnhdw** [comment, marked as answer]
`M1` stands for `Mouse Button 1` which translates to the left-click. It’s standard syntax used in many games.

**Reply by SparkyDabs** [reply]
I was in fact, think way too hard about it. lol Thank you so much *facepalm*
